src/Symfony/Component/PropertyInfo/Tests/Fixtures/NullExtractor.php

```diff
@@ -28,40 +28,58 @@ class NullExtractor implements PropertyListExtractorInterface, PropertyDescripti
      */
     public function getShortDescription($class, $property, array $context = array())
     {
+        $this->assertIsString($class);
+        $this->assertIsString($property);
     }
 
     /**
      * {@inheritdoc}
      */
     public function getLongDescription($class, $property, array $context = array())
     {
+        $this->assertIsString($class);
+        $this->assertIsString($property);
     }
 
     /**
      * {@inheritdoc}
      */
     public function getTypes($class, $property, array $context = array())
     {
+        $this->assertIsString($class);
+        $this->assertIsString($property);
     }
 
     /**
      * {@inheritdoc}
      */
     public function isReadable($class, $property, array $context = array())
     {
+        $this->assertIsString($class);
+        $this->assertIsString($property);
     }
 
     /**
      * {@inheritdoc}
      */
     public function isWritable($class, $property, array $context = array())
     {
+        $this->assertIsString($class);
+        $this->assertIsString($property);
     }
 
     /**
      * {@inheritdoc}
      */
     public function getProperties($class, array $context = array())
     {
+        $this->assertIsString($class);
+    }
+
+    private function assertIsString($string)
+    {
+        if (!is_string($string)) {
+            throw new \Exception(sprintf('"%s" expects strings, given "%s".', __CLASS__, gettype($string)));
```

Comment:
It doesn't matter that much as it's a fixture but it's indeed more adapted here.
updated :)